src/dmd/root/port.d

```diff
@@ -128,9 +128,9 @@ extern (C++) struct Port
     }
 
     // Little endian
-    static uint readlongLE(scope void* buffer) pure
+    static uint readlongLE(scope const(void*) buffer) pure
```

Comment:
Just `scope const void* buffer` would be fine. No need for the ( ). Ditto for the rest.